.github/workflows/tests.yml

```diff
@@ -13,7 +13,7 @@ jobs:
       fail-fast: false
       matrix:
         cache_nonce: [ 1 ]
-        nim_version: [ 1.6.14 ]
+        nim_version: [ 1.6.16 ]
```

Comment:
16 is buggy, I think 18 is out now?

Reply:
https://nim-lang.org/blog/2023/12/19/versions-1618-202-released.html

Reply:
Thanks for the info, fixing it